src/Frontend/Modules/Blog/Tests/Actions/CategoryTest.php

```diff
@@ -12,8 +13,7 @@ public function testCategoryHasPage(): void
         $this->loadFixtures(
             $client,
             [
-                'Backend\Modules\Blog\DataFixtures\LoadBlogCategories',
-                'Backend\Modules\Blog\DataFixtures\LoadBlogPosts',
+                LoadBlogPosts::class,
```

Comment:
don't you need to load the categories too?

Reply:
No, the LoadBlogPost will insert a category as well.